### ISSUE TITLE
some repos missing

### ISSUE THREAD

**Issue by philliproberts-BH** [Capgemini-test-import/inner-source-backend-repos#9]
I visited the [Inner Source Portal](https://github.build.ge.com/pages/OSPO/inner-source-frontend/#display=card) and noticed that only one out of 3 repos in [Baker Hughe AI organization](https://github.build.ge.com/bh-ai) are featured in the page.
I wonder if this is due to some kind of bug related to pagination or access rights.

**Comment by philliproberts-BH** [comment]
The reason why two repositories weren't listed is that they are set to "internal" visibility instead of "public". I will change their visibility then.

**Comment by philliproberts-BH** [comment]
I actually think this was due to incorrectly creating the token. After some investigation, I learned only "public" repos were visible. After recreating the token, all repos (public, internal, private) are now visible.

That being said, the scanner is not currently being ran regularly. So if anyone else notices a newer repo not showing, this is probably because we haven't ran it lately. After the migration in September, the scanner(s) should run daily.
